lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -89,6 +89,11 @@ e2function number entity:id()
 	return this:EntIndex()
 end
 
+e2function number entity:creationID()
+	if not IsValid(this) then return 0 end
```

Comment:
I'd prefer it to match the rest of the file personally, and allows for better readability in my own opinion. 

A person new to Lua trying to figure out what a function does may be confused by reading that.